ParabolInc/parabol
# feat(suggest-groups): add loading feedback while waiting for OpenAI response

Fix https://github.com/ParabolInc/parabol/issues/8133

Demo: https://www.loom.com/share/47d247ba6dce446fa22917f79f4cc542

### To test

- Add the Suggest Groups org feature flag
- Add a bunch of reflections and proceed to the Group phase
- See that the Suggest Groups button is disabled until `autogroupReflectionGroups` exists in our db
